cobalt/android/BUILD.gn

```diff
@@ -155,6 +155,41 @@ template("content_shell_apk_tmpl") {
   }
 }
 
+# Copied from target: libcontent_shell_content_view
+# TODO rename this library, NO shell
```

Comment:
`TODO(cobalt)` or some other way to make this more grep-able please :)

Reply:
TODO(b/123): bug format. We can put a linter in for this if needed.